helm-repositories/banzaicloud/banzaicloud.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: source.toolkit.fluxcd.io/v1beta1
 kind: HelmRepository
 metadata:
   name: kubernetes-charts.banzaicloud.com
+  namespace: ${WORKSPACE_NAMESPACE}
```

Comment:
should this be `${workspaceNamespace}`?

Reply:
oops you're right, it should be `${workspaceNamespace}`

Reply:
Done